src/app/sidenav/sidenav.component.html

```diff
@@ -4,6 +4,8 @@
     <mat-nav-list>
       <a mat-list-item [routerLink]="['/create-community']">Create Community</a>
       <mat-divider></mat-divider>
+      <a mat-list-item [routerLink]="['/creation']">Create Memes</a>
```

Comment:
We should only display this if they're logged in, `*ngIf="userService.isLoggedIn()"`

Reply:
I disagree with that, someone who has never signed up might not even know about the meme creation functionality if the button isn't accessible. If anything they should be allowed to create a meme, and be prompted to log in/sign up as soon as they try to publish/download